pysces/panel.py

```diff
@@ -46,6 +50,15 @@ def _update(self, Uinfty=(1,0)):
         self._vortices = Vortices(xvort)
         self._influence_matrix = None
 
+        # Uncomment the following lines to see the vortex and collocation
```

Comment:
Looks to me as though this is the only place matplotlib is needed.  Could put "import matplotlib.pyplot as plt" here (commented out) if you want to leave in this plotting stuff, and then we still wouldn't depend on matplotlib.